### PR TITLE
Remove "View/Func" dependency

### PR DESCRIPTION
**Pull request by stephencelis**
We talked about getting our "Html" version up-to-date and eliminating this abstraction/dependency seems like a good first step. Started this last night. Mostly a mindless task but took longer than I expected.

### CODE REVIEW

File: Sources/PointFree/Admin/FreeEpisodes.swift [hunk 1]
```diff
@@ -52,7 +51,7 @@ private func sendEmail(forFreeEpisode episode: Episode, toUsers users: [User]) -
 
   // A personalized email to send to each user.
   let freeEpisodeEmails = users.map { user in
-    lift(IO { inj2(freeEpisodeEmail.view((episode, user))) })
+    lift(IO { inj2(freeEpisodeEmail((episode, user))) })
```
Comment (stephencelis) [inline Sources/PointFree/Admin/FreeEpisodes.swift]: Email views are still pullback-composed, so the tupling prevails.

File: Sources/PointFree/Account/Cancel.swift [hunk 2]
```diff
@@ -207,44 +206,45 @@ private func sendCancelEmail(to owner: User, for subscription: Stripe.Subscripti
...
+    newsletter: nil,
+    title: "Your subscription has been canceled",
+    preheader: """
+    Your \(subscription.plan.nickname) subscription has been canceled and will remain active through
+    \(dateFormatter.string(from: subscription.currentPeriodEnd)).
+    """,
+    template: .default,
+    data: (owner, subscription)
+  )
+  } >>> simpleEmailLayout(cancelEmailBodyView)
```
Comment (mbrandonw) [inline Sources/PointFree/Account/Cancel.swift]: super nitty, but what do you think about doing `simpleEmailLayout(cancelEmailBodyView) <<<` at the beginning instead? kinda of nice to have the layout at the top

Comment (stephencelis) [reply Sources/PointFree/Account/Cancel.swift]: Can do. Just was `>>>`ing everything.

File: Sources/PointFree/Account/PrivateRss.swift [hunk 3]
```diff
@@ -159,11 +158,13 @@ private func fetchStripeSubscriptionForUser<A>(
     }
 }
 
-private let privateEpisodesFeedView = itunesRssFeedLayout <| View<(Stripe.Subscription?, User)> { subscription, user -> Node in
-  node(
-    rssChannel: privateRssChannel(user: user),
-    items: items(forUser: user, subscription: subscription)
-  )
+private let privateEpisodesFeedView = itunesRssFeedLayout { (data: (subscription: Stripe.Subscription?, user: User)) -> [Node] in
```
Comment (mbrandonw) [inline Sources/PointFree/Account/PrivateRss.swift]: oh interesting. I never thought about using this syntax for a layout. could we even replace the other `layout <<< { ... }`'s with just `layout { ... }` ?

Comment (stephencelis) [reply Sources/PointFree/Account/PrivateRss.swift]: We'd need to un-curry `simpleEmailLayout`. Worth doing but let's kick to another PR.

File: Sources/PointFree/Admin/FreeEpisodes.swift (hunk 1 above)
Comment (mbrandonw) [reply Sources/PointFree/Admin/FreeEpisodes.swift]: yeahhh

**Comment by mbrandonw** [comment]
oh I guess the `Package.swift` could be updated too?

**Comment by stephencelis** [comment]
So nice to have snapshot tests! One gnarly part of the refactor got caught. There may be more, but as soon as all the snaps pass will merge!

**Comment by mbrandonw** [comment]
Wow very nice to catch that footer bug.

**Pull request merged**